src/main/kotlin/eu/kevin/api/Client.kt

```diff
@@ -0,0 +1,46 @@
+package eu.kevin.api
+
+import io.ktor.client.*
+import io.ktor.client.features.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+
+class Client internal constructor(
+    private val authorization: Authorization,
+    private val version: EndpointVersion,
+    private val httpClient: HttpClient
+) {
+    constructor(
+        authorization: Authorization,
+        version: EndpointVersion
+    ) : this(
+        authorization = authorization,
+        version = version,
+        httpClient = Dependencies.httpClient
```

Comment:
Does it make sense to use a DI library, since this SDK will be used as a library? There are Koin features like https://insert-koin.io/docs/reference/koin-core/start-koin#loading-modules-after-startkoin, but they require that the application would be using Koin themselves.

Reply:
After you have mentioned that this will be used as a library then it does not make sense to use any DI library inside. Resolving this.